packages/components/CONTRIBUTING.md

```diff
@@ -292,6 +292,43 @@ All components, in addition to being typed, should be using JSDoc when necessary
 
 Each component that is exported from the `@wordpress/components` package should include a `README.md` file, explaining how to use the component, showing examples, and documenting all the props.
 
+##### Readme example
+
+```
+# ComponentName
+
+If component is experimental, add the following section:
+<div class="callout callout-alert">
+This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
+</div>
+
+If component is deprecated, add the following section:
+<div class="callout callout-alert">
+This component is deprecated. Please use  `{other component}` from the `{other package}` package instead.
+</div>
+
+Description of the component.
+
+## Usage
+
+Code example using correct markdown syntax and formatted using project's formatting rules.
```

Comment:
```suggestion
Code example using correct markdown syntax and formatted using project's formatting rules. See [ItemGroup](/packages/components/src/item-group/item-group/README.md#usage) for a real-world example.

```jsx
import { ExampleComponent } from '@wordpress/components';

function Example() {
	return (
		<ExampleComponent>
			<p>Code is poetry</p>
		</ExampleComponent>
	);
}
```